### DISCUSSION TITLE
[Esame Gennaio 2020] Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#230] [Esami - Bertini @ Informatica]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/36181cdf-8707-4bbe-9ead-44d3faa58bca)

**Comment by LeoCosta02** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/b530e770-55b9-4276-bff1-e5a67eb2f10f)
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/7130482c-1dfc-4bd1-b599-bc5ff6bef459)

**Comment by cornflexxx** [comment]
@LeoCosta02 

Nel terzo punto il passaggio a complemento che hai fatto sta considerando anche i casi in cui ci sono due semi uguali ed uno diverso, ho trovato più facile svolgerlo direttamente:

$$P\\{\text{tre carte estratte con semi diversi}\\}=\frac{40 \cdot 30 \cdot 20}{40\cdot 39 \cdot 38}\approx 0.40$$

Ci servono tutte quelle tuple del tipo $(D,B,S)$ dove le lettere rappresentano il seme della carta.

---------------------

Allo stesso modo nel terzo punto non ho ragionato in termini di probabilità condizionata ma sempre in termini di casi favorevoli su casi possibili.
Casi favorevoli: sono tutti quei casi del tipo $(A_D,X,X)$, quindi il primo fisso e gli altri che permutano perciò $CF=\binom{39}{2}2!$
Casi possibili: sono tutti quei casi dello stesso tipo precedente ma i tre elementi permutano tutti quindi  $CP=\binom{39}{2}3!$

$$P\\{\text{asso al primo posto}\\}=CF/CP=\frac{1}{3}$$

**Comment by LeoCosta02** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/0e0c79bb-939a-49b7-af7c-f23e357c707a)
 @cornflexxx Adesso anche con il complemento dovrebbe essere giusto e poi ho calcolato come hai fatto tu in modo diretto

**Comment by LeoCosta02** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/a2d635b3-32bf-401c-97da-36b69214f9a7)
 Il quarto punto se si vuole usare la prob condizionata

**Comment by Elia-Belli** [comment]
**(1)** Lo spazio di probabilità dell'estrazione senza rimpiazzo di $3$ carte dalle $40$ è descritto come:
$$\Omega =\\{(w_1,w_2,w_3)| w_i=1,..,40\ w_i\neq w_j\\}$$ con $\mathbb{P}$ uniforme su $\Omega$ , dato che $|\Omega|=\binom{40}{3}$ ogni tupla ha probabilità $1/\binom{40}{3}$ di essere ottenuta.

**(2)** Possiamo avere una napoletana di $4$ semi differenti, quindi calcoliamo la probabilità per un seme e la moltiplichiamo per il numero di semi:
$$\mathbb{P}(napoletana)=4\cdot \mathbb{P}((1D,2D,3D))=\frac{4}{\binom{40}{3}}$$

**(3)** Possiamo scegliere i semi invece delle carte in $\binom{4}{3}=4$ modi diversi e per ogni seme scegliere una carta tra le $10$:

$$\mathbb{P}(semi\ diversi)=\frac{4\cdot 10^3}{\binom{40}{3}}\simeq 0.405$$

**(4)** Lo spazio $\Omega$ definito non considera l'ordine delle estrazioni, preferisco utilizzare la probabilità condizionata e ragionare sulle estrazioni con l'asso di denari favorevoli contro quelle possibili:

$$\mathbb{P}(Asso\ Denari\ 1°\ estratto)=\mathbb{P}(i=1|w_i=1D)=\frac{\mathbb{P}(i=1\cap w_i=1D)}{\mathbb{P}(w_i=1D)}=\frac{1/40}{3/40}=\frac{1}{3}$$

**Reply by CasuFrost** [reply]
Scusa una cosa non mi è chiara nel punto 3, scrivi che $\mathbb{P}(\omega_i=1D)=\frac{3}{40}$, ma non dovrebbe essere : 
$$\frac{1}{40}+\frac{1}{39}+\frac{1}{38}$$ 
?

**Comment by CiottoloMaggico** [comment]
![immagine](https://github.com/user-attachments/assets/e7232565-d982-4116-be78-f40147e8b330)
